examples/integrations/cdi/datasource-hikaricp/README.adoc

```diff
@@ -0,0 +1,108 @@
+= Hikari Connection Pool Integration Example
+
+== Overview
+
+This example shows a trivial Helidon MicroProfile application that
+uses the Hikari connection pool CDI integration, shows how to package
+it into a Docker image, and shows how to run it in a Docker container.
+It also shows how to run the Oracle database in a Docker container and
+connect to it using the application's container.
+
+== Notes
+
+To create a user-defined Docker network:
+
+```sh
+$ docker network create helidon-examples
+```
+
+To log in to the Oracle Container Registry (which you will need to do
+in order to pull Oracle database images from it):
+
+```sh
+$ docker login -u your.name@yourdomain.com -p yourpasswordhere container-registry.oracle.com
+```
+
+To run Oracle's `database/standard` Docker image in a Docker container
+named `mysql` connected to this network that publishes its ports to
+the host:
+
+```sh
+$ docker container run --name oracle --network helidon-examples --detach -it --publish 1521:1521 --publish 5500:5500 --shm-size=3g container-registry.oracle.com/database/standard:latest
+```
+
+It will take about ten minutes before the database is ready.
```

Comment:
10 minutes? Unbelievable :)

Reply:
I've used the oracle db in a container using a different image (`store/oracle/database-enterprise:12.2.0.1-slim`), it does indeed take a while to start and it's not obvious how to tell when the database is effectively ready.